### ISSUE TITLE
Prolem when create page from fuel admin page

### ISSUE THREAD

**Issue by sontq** [daylightstudio/FUEL-CMS#601]
Dear team,

Please help me, when i create page from fuel admin (lastest version) successfull. But, after view page body content have remove some html tag (input tag, button tag, i tag html icon...). How to create page content include input tag, button tag, i tag html icon ... from admin page ?

Thanks team

**Comment by daylightstudio** [comment]
That sounds like something that the HTML purifier may be stripping out as a default security setting. Try changing the `fuel/application/config/purifier.php` by updating the `HTML.allowed` to include the tags you need to enter in.

**Comment by sontq** [comment]
> That sounds like something that the HTML purifier may be stripping out as a default security setting. Try changing the `fuel/application/config/purifier.php` by updating the `HTML.allowed` to include the tags you need to enter in.

I changed config HTML.allowed but HTML purifier not support input tag, div tag data attribute. And html font awesome still removed (<i class="fa fa-phone"></i>). Please help me check.

Thanks team

**Comment by daylightstudio** [comment]
There is an "enabled" parameter in the `fuel/application/config/purifier.php` that you can set to `FALSE`.

**Comment by sontq** [comment]
> There is an "enabled" parameter in the `fuel/application/config/purifier.php` that you can set to `FALSE`.

Dear team,

I have disable purifier but html tag i not content inside ( icon css fontawesome ) still removed.

**Comment by daylightstudio** [comment]
What is the HTML code you are trying to put in your page?

**Comment by sontq** [comment]
I use html icon font awaresome at here https://fontawesome.com/v4/icons/

**Comment by daylightstudio** [comment]
Right, but what is the code you are entering in the Pages module that is getting stripped out... and is it in the "Body" field or some other template field?

**Comment by sontq** [comment]
![Screenshot](https://user-images.githubusercontent.com/4971199/207231675-19f070d3-ec42-463c-8fc2-c983ca4768b2.png)
Here, I put html code in body field. I disable purifier but html tag i (red rectangle in screenshot) still removed after save page.

**Comment by daylightstudio** [comment]
I'm having a hard time replicating that problem. Can you paste in the text you are using in the screenshot here?

**Comment by sontq** [comment]
[test.txt](https://github.com/daylightstudio/FUEL-CMS/files/10219905/test.txt)
Please help me check content in test.txt file. I can't send content from git editor because git remove html tag in content.

Thanks

**Comment by daylightstudio** [comment]
Try using the add code button:
<img width="72" alt="image" src="https://user-images.githubusercontent.com/410984/207386302-4fcf9b76-66fd-411d-8433-92e1098d9d42.png">

**Comment by sontq** [comment]
I can't see button in tool bar. But i use source mode parse html code. Please check screenshot
![Screenshot](https://user-images.githubusercontent.com/4971199/207389142-cee849a1-4df7-4282-ac2c-7c930d905fcd.png)

**Comment by sontq** [comment]
Hi this is html code `<i class="fa fa-download"></i>`

**Comment by daylightstudio** [comment]
Can you send all the text you have in that body field? It may be CKEditor stripping things out. You can test by changing the `text_editor` config value in `fuel/application/config/MY_fuel.php` to the following to see if that is the problem:
```
$config['text_editor'] = 'markitup';
```

**Comment by sontq** [comment]
> Can you send all the text you have in that body field? It may be CKEditor stripping things out. You can test by changing the `text_editor` config value in `fuel/application/config/MY_fuel.php` to the following to see if that is the problem:
> 
> ```
> $config['text_editor'] = 'markitup';
> ```

Hi team,

I try change config editor to markitup and every ok.

Thanks a lot.
